lineapy/graph_reader/node_collection.py

```diff
@@ -142,7 +144,7 @@ def _update_graph(self, graph: Graph) -> None:
         """
         self.graph_segment = graph.get_subgraph_from_id(list(self.node_list))
```

Comment:
graph_segment is used elsewhere (node_collection does not have access to the full graph, only node_list) so keeping the update here.